types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -22,7 +22,9 @@ export interface BaseChatItemProps {
 
 export interface ChatItemMyTextProps extends BaseChatItemProps {
   status?: ChatStatus;
-  chatType?: string;
+  chatType: 'SIGNED' | 'ANONYMOUS';
+  messageType: string;
+  data: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `chatType` property has been changed from optional to mandatory and its type has been restricted to 'SIGNED' or 'ANONYMOUS'. This is a good change for type safety if these are the only valid values. However, it might break existing code that does not provide this property or provides different values. Make sure all usage of this interface is updated accordingly.

The `messageType` and `data` properties have been added. Ensure that they are always provided where this interface is used, as they are not optional. Also, using `any` as a type for `data` defeats the purpose of TypeScript's static typing. Consider replacing `any` with a more specific type if possible.
```diff
-  chatType?: string;
+  chatType: 'SIGNED' | 'ANONYMOUS';
+  messageType: string;
+  data: any;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->